admin/test/test_acceptance.py

```diff
@@ -205,3 +208,27 @@ def test_non_json_message(self):
             )],
             self._convert(NON_JSON_JOURNAL_EXPORT),
         )
+
+
+class CommonOptionsTests(TestCase):
+    """
+    Tests for ``CommonOptions``.
+    """
+    def test_invalid_version(self):
+        """
+        An ``UsageError`` is thrown if an invalid flocker version is passed
+        to the ``CommonOptions``.
+        """
+        arg_options = (
+            "--distribution ubuntu-14.04 --provider AWS "
+            "--flocker-version invalid-version"
+        )
```

Comment:
:question: Personally, I'd make this a tuple rather than splitting it later. Imaging if one of the option values contained whitespace.